seedcase_sprout/core/sprout_checks/get_polars_data_type.py

```diff
@@ -0,0 +1,52 @@
+import polars as pl
+
+from seedcase_sprout.core.properties import FieldType
+
+
+def get_polars_data_type(field_type: FieldType | None) -> pl.DataType:
+    """Maps Frictionless field types to Polars data types.
+
+    If the Frictionless field type has formatting constraints that are not included
+    in any specialised Polars data type, the mapping is to string. The formatting
+    constraints are then checked without Polars.
+
+    Args:
+        field_type: The Frictionless field type to map.
+
+    Returns:
+        The Polars data type the field is mapped to.
+
+    Raises:
+        NotImplementedError: If Sprout doesn't yet support the Frictionless field type.
+    """
+    match field_type:
+        case "geojson":
+            raise NotImplementedError()
+        # While Polars does have all these data types, there isn't a
```

Comment:
```suggestion
        # While Polars does have most of these data types, there isn't a
```
Because year, yearmonth, geopoint don't have matches